xml/System.Windows.Forms/DataGridViewRowStateChangedEventArgs.xml

```diff
@@ -82,6 +80,7 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentNullException">.NET 6 and later versions: <see paramref="dataGridViewRow" /> is <see langword="null" />.</exception>
```

Comment:
This doesn't appear to render correctly:
![image](https://user-images.githubusercontent.com/4403806/112931909-bab54400-9168-11eb-8317-fab5e075b914.png)

Reply:
Thanks - good catch!